tests/UsedClassesAvailableTest.php

```diff
@@ -1,6 +1,6 @@
 <?php declare(strict_types=1);
 
-namespace Swag\CustomCmsElementTests;
+namespace Swag\CustomCmsElement\tests;
```

Comment:
please add a proper namespace to the composer.json and change this Namespace to `Swag\CustomElement\Test` like here https://github.com/shopwareLabs/SwagPayPal/blob/master/composer.json#L40-L43

Reply:
Done